### ISSUE TITLE
Question regarding gen9 outbreak events

### ISSUE THREAD

**Issue by observeroftime01** [projectpokemon/EventsGallery#382]
Hey, great to see that these are being archived. Is there a tangible use for the outbreak files yet? How do I import them into my game?

I see that PKHeX [recently ](https://github.com/kwsch/PKHeX/commit/07191e6aadc69b015e7cf47ef4b24d24ea41e153)added a lot of new outbreak blocks to the program. I have no success importing pokedata_array_2_0_0, zone_main_array_2_0_0, and zone_su1_array_2_0_0 into KBCATOutbreakPokeData, KBCATOutbreakZonesPaldea, and KBCATOutbreakZonesKitakami respectively. I mean, the files import just fine being the right size for their block and everything, but nothing shows up in-game after time-skipping a bunch. Is there anything I'm missing? Too early?

**Comment by foohyfooh** [comment]
The likely reason is that you didn't change the enable block. You can follow https://projectpokemon.org/home/tutorials/save-editing/gen-9/gen-9-specific-edits-importing-mass-outbreak-events-r127/ to see what to do.

**Comment by observeroftime01** [comment]
Cheers, I did not see the tutorial when I looked over the information on the front page of this repository. This explains things perfectly. I think I even looked over information at projectpokemon.org regarding importing events from this gallery, and didn't see it at the time. Is this a recent addition or has it been there for a while? 

If I may suggest, if it hasn't already been included, it might be worth including a link to the tutorial in the repository readme, in case someone else like me comes along, haha.

Thanks for the answer, question (topic) can be closed 👍
